vue-frontend/src/components/home-new/CTASection.vue

```diff
@@ -0,0 +1,219 @@
+        How do you know we’re a good fit for you?
+      </div>
+      <div class="v2-title-3-text mt-4">Let’s have a short chat.</div>
+      <div class="communication">
+        <div class="item px-4 py-3">
+          <div class="v2-normal-3-text">I love talking.</div>
+          <button class="gradient-btn cta-button" @click="openContact">
+            <font-awesome-icon
+              class="fa v2-title-3-text"
+              icon="phone"
+              aria-hidden="true"
+            />
+            <span class="v2-normal-3-text">Schedule a call</span>
+          </button>
+        </div>
+        <div class="item px-4 py-3">
+          <div class="v2-normal-3-text">I prefer mail.</div>
+          <button class="gradient-btn cta-button" @click="openContact">
+            <font-awesome-icon
+              class="fa v2-title-3-text"
+              icon="paper-plane"
+              aria-hidden="true"
+            />
+            <span class="v2-normal-3-text">Drop a mail</span>
+          </button>
+        </div>
+      </div>
+      <div
+        class="v2-title-3-text text-left v2-canopas-gradient-text reason-text mt-5"
+      >
+        Four reasons why do we do this call?
+      </div>
+      <ul class="v2-normal-2-text chat-list text-left mt-4">
+        <li>
+          <span class="bullet"></span>
+          <div>It gives an opportunity to learn more about each other.</div>
+        </li>
+        <li>
+          <span class="bullet"></span>
+          <div>
+            We want to give you a chance to put your idea into action. You might
+            have faced this situation before where you wish you had someone to
+            help you.
+          </div>
+        </li>
+        <li>
+          <span class="bullet"></span>
+          <div>
+            Sometimes, you want to check the feasibility of your idea. We will
+            give you honest feedback based on our industry experience.
+          </div>
+        </li>
+        <li>
+          <span class="bullet"></span>
+          <div>
+            As you read above, we offer 100% Money-back guarantee. so, there’s
+            no risk to you.
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script type="module">
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import backgroundImage from "@/assets/images/theme/cta-background.webp";
+
+export default {
+  data() {
+    return {
+      backgroundImage: backgroundImage,
+    };
+  },
+  components: {
+    FontAwesomeIcon,
+  },
+  methods: {
+    openContact() {
+      this.$router.push("/contact");
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.background {
+  background: #3d3d3d;
+  padding: 100px 0;
+  overflow: hidden;
+  border-radius: 100px 0;
+  position: relative;
+  z-index: 0;
+}
+
+.render-background {
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  position: absolute;
+  object-fit: cover;
+  z-index: -1;
+}
+
+.container {
+  width: 95%;
```

Comment:
Why do we need to change the container width as it is a bootstrap class having width?

Reply:
Because in mobile devise not look properly

Reply:
You can adjust padding instead of width